apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/McsClaimTransformerV2.java

```diff
@@ -274,7 +277,8 @@ private static Money getTotal(PreAdjMcsClaim claimGroup) {
                       .setSequence(detail.getPriority() + 1)
                       .setProductOrService(
                           new CodeableConcept(
-                              new Coding(BBCodingSystems.HCPCS, detail.getIdrProcCode(), null)))
+                              new Coding(
+                                  BBCodingSystems.MCS.PROC_CODE, detail.getIdrProcCode(), null)))
```

Comment:
Per a conversation with Bill Harty, lets use "https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets" as the system for MCS HCPCS codes